src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax;
+
+public class Main {
+    public static void main(String[] args) {
+        ColorSupplier colorSupplier = new ColorSupplier();
+        Figure[] figures = new Figure[10];
+
+        FigureSupplier figureSupplier = new FigureSupplier();
+
```

Comment:
redundant blank lines
```suggestion
        FigureSupplier figureSupplier = new FigureSupplier();
```

Reply:
not fixed